lib/bgsite_official_web/templates/admin_controller/index.html.eex

```diff
@@ -0,0 +1 @@
+<h1> Hello </h1>
```

Comment:
The templates directory is only for templates :), remove admin_controller inside templates (and admin_controller/index)

Reply:
I agree with @ykostov :+1: